### PR TITLE
59

### PR DESCRIPTION
**Pull request by praveen-betalectic**

### CODE REVIEW

File: src/stories/Admin/CanDeleteAdmin/story.js [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+  }
+};
+
+const handle = async ({ prepareResult, authorizeResult }) => {
+  try {
+    let payload = {
+      email: prepareResult.email,
+      password: prepareResult.password,
+    };
+    return await adminRepo.createAdmin(payload);
```
Comment (rjvim) [inline src/stories/Admin/CanDeleteAdmin/story.js:47]: @praveen-betalectic The story is canDeleteAdmin, but we are calling create Admin right?

**Review by rjvim — CHANGES_REQUESTED** [review]

File: src/stories/Admin/CanDeleteAdmin/story.js (hunk 1 above)
Comment (praveen-betalectic) [reply src/stories/Admin/CanDeleteAdmin/story.js]: @rjvim code changed please review again.

Comment (rjvim) [reply src/stories/Admin/CanDeleteAdmin/story.js]: @praveen-betalectic There is a postman collection in repo; please import that into your Postman add new endpoints also and update it back into repo.

Comment (praveen-betalectic) [reply src/stories/Admin/CanDeleteAdmin/story.js]: @rjvim postman collection updated.

**Pull request merged**